examples/build/Makefile

```diff
@@ -45,6 +45,9 @@ de0nano: #altera/
 de0cv: #altera/
 	$(PYTHON) ex_de0cv.py
 
+de0nano: #altera/
```

Comment:
don't have two targets of the same, if you want to run the de1soc under the de0nano target add as additional command under de0nano.